WORKSPACE

```diff
@@ -94,14 +110,14 @@ npm_translate_lock(
         "@gregmagolan/test-a@0.0.1": ["//examples/npm_deps:patches/test-a@0.0.1.patch"],
     },
     pnpm_lock = "//:pnpm-lock.yaml",
-    pnpm_version = "6.32.19",
```

Comment:
curious, why remove that?

Reply:
did we want to do this?

Reply:
I think this is here to provide a bit of test coverage for that attribute so yeah, let's keep it and just update the version

Reply:
roger roger